src/components/MessageInput/MessageInputFlat.tsx

```diff
@@ -214,10 +213,10 @@ const MessageInputV2 = <
 
   const accept = useMemo(
     () =>
-      zipObject(
-        acceptedFiles,
-        Array.from({ length: acceptedFiles.length }, () => []),
-      ),
+      acceptedFiles.reduce<Record<string, Array<string>>>((av, cv) => {
```

Comment:
TIL about `??=` ;). Apart from that, I feel like the variable naming for `av` and `cv` can be improved to something that requires less mental overhead.